includes/data-stores/class-wcs-orders-table-subscription-data-store.php

```diff
@@ -343,23 +343,6 @@ public function create( &$subscription ) {
 		do_action( 'woocommerce_new_subscription', $subscription->get_id() );
 	}
 
-	/**
-	 * Reads multiple subscription objects from custom tables.
-	 *
-	 * @param \WC_Order $subscriptions Subscription objects.
-	 */
-	public function read_multiple( &$subscriptions ) {
```

Comment:
We were overriding this read_multiple function only to make sure we called `set_subscription_props()` after each subscription was read. This is no longer necessary because what `set_subscription_props()` use to do is now done via `set_order_props_from_data()` which is called during the read.